examples/example-golang/main.go

```diff
@@ -70,6 +71,9 @@ func main() {
 		}
 		if !wf.Status.FinishedAt.IsZero() {
 			fmt.Printf("Workflow %s %s at %v\n", wf.Name, wf.Status.Phase, wf.Status.FinishedAt)
+			if wf.Status.Phase == wfv1.NodeFailed || wf.Status.Phase == wfv1.NodeError {
+				fmt.Printf("Workflow %s failed due to: %s", wf.Name, wf.Status.Message)
```

Comment:
could you just add the `wf.Status.Message` to line 74?

Reply:
Sure updated.